app/views/shared/_flash.html.erb

```diff
@@ -0,0 +1,5 @@
+<% flash.each do |key, message| %>
+  <div class="<%= flash_class(key) %>" role="alert">
+    <%= message %>
+  </div>
+<% end %>
```

Comment:
I thought for sure we had this already -- thanks for writing this!

On lines 78-85 of `/app/views/layouts/application.html.erb` there's some flash code placed there already. Could you please:
 - Replicate that code here (it includes some Accessibility stuff)
 - Replace that code with rendering this partial 

Thank you!